data/Pakistan/Assembly/term-14.csv

```diff
@@ -191,7 +192,7 @@ cc6f31bb-70cf-4517-824b-981b15d9efa6,Muhammad Junaid Anwar Chaudhary,Muhammad Ju
 073ace5a-3f2e-43e3-a1a6-73b5e518384d,Muhammad Khan Daha,Muhammad Khan Daha,,,,PML(N),pmln,area/na-157_(khanewal-ii),NA-157 (Khanewal-II),National Assembly,14,,,http://www.na.gov.pk/uploads/images/Muhammad%20sKhan%20sDaha(1).jpg,male,Q18815970,Q799577,Q12812171
 aa87aa59-451e-48a3-b755-15becf80df53,Muhammad Moeen Wattoo,Muhammad Moeen Wattoo,,,,PML(N),pmln,area/na-147_(okara-v),NA-147 (Okara-V),National Assembly,14,,,http://www.na.gov.pk/uploads/images/147.JPG,male,Q18815998,Q799577,Q12856786
 0e2c4697-c9b4-4df4-a90a-98c7af328ed5,Muhammad Muzammil Qureshi,Muhammad Muzammil Qureshi,,,,MQM,mqm,area/na-253_(karachi-xv),NA-253 (Karachi-XV),National Assembly,14,,,http://www.na.gov.pk/uploads/images/253.jpg,male,Q19517997,Q1265113,Q12857475
-7304486c-f12d-4009-b2db-ee47ec4f5771,"Muhammad Nawaz Sharif, Prime Minister Islamic Republic of Pakistan","Muhammad Nawaz Sharif, Prime Minister Islamic Republic of Pakistan",,,,PML(N),pmln,area/na-120_(lahore-iii),NA-120 (Lahore-III),National Assembly,14,,,http://www.na.gov.pk/uploads/images/120.jpg,male,Q134068,Q799577,Q12780067
+7304486c-f12d-4009-b2db-ee47ec4f5771,"Muhammad Nawaz Sharif, Prime Minister Islamic Republic of Pakistan","Muhammad Nawaz Sharif, Prime Minister Islamic Republic of Pakistan",,,,PML(N),pmln,area/na-120_(lahore-iii),NA-120 (Lahore-III),National Assembly,14,,2017-07-28,http://www.na.gov.pk/uploads/images/120.jpg,male,Q134068,Q799577,Q12780067
```

Comment:
If we're removing the Prime Minister suffix from names (which is definitely a good idea!), we should probably remove it from here too.

Reply:
Good point, I've removed that from the archived source.